src/test/scala/com/github/simplesteph/ksm/compat/AdminClientAuthorizerTest.scala

```diff
@@ -0,0 +1,84 @@
+package com.github.simplesteph.ksm.compat
+
+import com.github.simplesteph.ksm.AclSynchronizer
+import com.github.simplesteph.ksm.notification.DummyNotification
+import com.github.simplesteph.ksm.parser.CsvAclParser
+import com.github.simplesteph.ksm.source.DummySourceAcl
+import net.manub.embeddedkafka.{EmbeddedKafka, EmbeddedKafkaConfig}
+import org.scalatest.{FlatSpec, Matchers}
+import org.apache.kafka.clients.admin.AdminClientConfig
+
+import scala.collection.JavaConverters._
+
+trait JaasConfiguration {
```

Comment:
not a huge fan of this. There's a `sasl.jaas.config` setting directly in the client we can use that's cleaner.

Also, I believe any jaas java property passed should automatically be picked up by the client. What's the intent of this?

Reply:
Totally agree it looks very weird. I've tried `sasl.jaas.config` first, but it didn't help.
As far as I remember, the problem was that we can't configure zookeeper in embedded-kafka, but it's still needed internally to be configured consistently with broker jaas.
If we don't want that in test code, I'll try investigate further

Reply:
Makes sense, didn't see it was part of the test suite. Keep it for now :)